frontend/src/Student/components/ShowCard.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
+import styled from 'styled-components'
+import { Link } from 'react-router-dom'
+import PropTypes from 'prop-types'
+import Moment from 'react-moment'
+import FaPlusCircle from 'babel-loader!react-icons/fa/plus-circle'
+import { Button, ButtonGroup, Row, Col } from 'reactstrap'
+
+const Card = styled.div`
```

Comment:
Visually, this feels too narrow. `min-height: 250px` seems to work well. You'll have to recenter the "New Submission" button.